go.sum

```diff
@@ -169,6 +169,7 @@ github.com/hashicorp/memberlist v0.1.0/go.mod h1:ncdBp14cuox2iFOq3kDiquKU6fqsTBc
 github.com/hashicorp/serf v0.0.0-20160124182025-e4ec8cc423bb/go.mod h1:h/Ru6tmZazX7WO/GDmwdpS975F019L4t5ng5IgwbNrE=
 github.com/hashicorp/terraform v0.12.0 h1:It2vmod2dBMB4+r+aUW2Afx0HlftyUwzNsNH3I2vrJ8=
 github.com/hashicorp/terraform v0.12.0/go.mod h1:Ke0ig9gGZ8rhV6OddAhBYt5nXmpvXsuNQQ8w9qYBZfU=
+github.com/hashicorp/terraform v0.12.1 h1:A56pUiGbDZ372cvDKWixzVciTGZkqZ01hH58GTNQ4lg=
```

Comment:
We only recently switch from dep to go modules, so I my knowledge is not strong here, but it seems that having two different versions of hashicorp/terraform here is sub-optimal.

Reply:
This is just the checksum, as I understand it the version used will still be from go.mod, it must have just pulled this in since I have it installed. There are a few other dependencies (e.g. aws-sdk) that also have multiple versions in the go.sum file.